core/src/test/java/edu/wpi/grip/core/PipelineRunnerTest.java

```diff
@@ -386,7 +388,7 @@ public synchronized void failed(Service.State from, Throwable failure) {
     @SuppressWarnings("PMD.SystemPrintln")
```

Comment:
Sure.  Weird that there is a PMD suppression because we do not even have PMD in the build (yet)...